resources/sass/custom.scss

```diff
@@ -20,3 +20,10 @@ footer {
   width: 100%;
   background-color: #800000;
 }
+
+#shopOwner-footer {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  background-color: #000066;
+}
```

Comment:
Since this same as footer for most of it, can we merge this and overide the background color only.

Reply:
yah that is better i guess.